### PR TITLE
smartplaylist: expose config as CLI options

### PR DESCRIPTION
**Pull request by mgoltzsche**
## Description

Add CLI options to `splupdate` command:
* `--playlist-dir`, `-d`
* `--relative-to`
* `--prefix`
* `--urlencode`
* `--forward-slash`
* `--pretend-paths`

## To Do

- [x] Documentation. (If you've added a new command-line flag, for example, find the appropriate page under `docs/` to describe it.)
- [x] Changelog. (Add an entry to `docs/changelog.rst` to the bottom of one of the lists near the top of the document.)
- [x] ~~Tests~~. (Very much encouraged but not strictly required.)

### CODE REVIEW

**Comment by Serene-Arc** [comment]
looks good to me

**Comment by mgoltzsche** [comment]
I thought documenting the CLI options explicitly would not be necessary since the corresponding configuration fields are already documented. Maybe I can add a line at the end of the "Configuration" section mentioning that there's a CLI option for each config option and that CLI options take precendence over config options...

File: beetsplug/smartplaylist.py [hunk 1]
```diff
@@ -72,16 +72,47 @@ def commands(self):
...
+            help="prepend string to every path in the playlist file.",
+        )
+        spl_update.parser.add_option(
+            "--forward-slash",
+            action="store_true",
+            dest="forward_slash",
+            help="Force forward slash in paths within playlists.",
+        )
+        spl_update.parser.add_option(
+            "--urlencoded",
```
Comment (mgoltzsche) [inline beetsplug/smartplaylist.py:102]: I realize this is wrong: it must called `urlencode` but wrongly documented!

**Comment by mgoltzsche** [comment]
I made some changes now:
* Added a paragraph after the configuration option list mentioning the CLI options.
* Also exposed the `--pretend-paths` option within the CLI.
* Replaced `urlencoded` with `urlencode` within the docs to reflect the actual option name.

Please re-review!

**Comment by Serene-Arc** [comment]
Excellent, thank you!

**Pull request merged**